src/java.desktop/macosx/classes/com/apple/laf/AquaFileChooserUI.java

```diff
@@ -2053,7 +2056,7 @@ int getApproveButtonMnemonic(final JFileChooser fc) {
 
         // No fallback
         String getApproveButtonToolTipText(final JFileChooser fc) {
-            return getApproveButtonToolTipText(fc, null);
+            return getApproveButtonToolTipText(fc,chooseButtonToolTipText);
```

Comment:
```suggestion
            return getApproveButtonToolTipText(fc, chooseButtonToolTipText);
```
There was a space after the comma as it should be, let's preserve it there.